components/DetailsPageHighlights/detailspagehighlights.scss

```diff
@@ -0,0 +1,183 @@
+.bx--artifact-details-highlight-container {
+
+  & .bx--createArtifactFlexGroupColumn {
+    margin-bottom: rem(35px);
+  }
+
+  & .bx--createArtifactFlexGroupColumn {
+    display: flex;
+    flex-direction: column;
+
+    &.bx--hide {
+      display: none;
+    }
+  }
+
+  & .bx--title-label {
+    font-size: 1.55rem;
+    font-weight: 200;
+    line-height: 1.5;
+  }
+
+  & .bx--description-label {
+    font-size: rem(14px);
+    font-weight: 300;
+    margin: rem(25px) 0;
+  }
+
+  @include breakpoint(825px) {
```

Comment:
any media query should be embedded in the class that needs it. Additionally (this is new), we're moving away from the `@include breakpoint` to a more standard

`@media (min-width: breakpoint('xs'))`

with the mapping looking like

```
$grid-breakpoints: (sm: 576px, md: 768px, lg: 992px, xl: 1200px, xxl: 1600px);
```